ovos-i2csound

```diff
@@ -272,7 +290,12 @@ main() {
         amixer -c "aiyvoicebonnet" cset numid=47 on
         amixer -c "aiyvoicebonnet" cset numid=49 on
         amixer -c "aiyvoicebonnet" cset numid=27 on
+        i2c_device_name="AIYVOICEBONNET"
+    fi
 
+    # Write the detection results to a file only if something is detected
+    if [ "$i2c_device_name" ]; then
+        echo "$i2c_device_name" > /home/ovos/.config/mycroft/i2c_platform
```

Comment:
Does not check for directory existence. Could this use an envvar or something to be configurable? Alternatively, a path in `/opt` or `/etc` wouldn't depend on a particular username